src/api/public/apidocs-new/paths/request_request_id.yaml

```diff
@@ -185,3 +185,50 @@ post:
                 summary: Couldn't find request with id '120'
   tags:
     - Request
+
+put:
+  summary: Modify a given request.
+  description: |
+    Modify a given request. **NOTE:** Certain parts of a request can only be changed by admins.
```

Comment:
We should add the reference to the schema: "XML Schema used for body validation: request.rng".